### ISSUE TITLE
Additional Processing for Sentinel-1 Data?

### ISSUE THREAD

**Issue by patrickcgray** [sentinel-hub/sentinelhub-py#66]
I've successfully downloaded sentinel-1 thanks to the really helpful jupyter notebook examples but I'm curious what kind of post-processing needs to go into the data now? 

I've ingesting this into a series of Keras deep learning and SKlearn machine learning models for land cover classification and I'm just not sure what state the data is in and what additional processing might be necessary. Are there any good examples of this or somewhere in the docs I can read to learn more?

Thanks for all the work thus far!

**Comment by gmilcinski** [comment]
This page describes the state of the data:
https://sentinel-hub.com/develop/documentation/eo_products/Sentinel1EOproducts
You might want to do some speckle filtering.
Our team did not do much with S1 in terms of machine learning. So I hope some other user will contribute their knowhow as well.

**Comment by patrickcgray** [comment]
Thanks @gmilcinski for the reply. So is there any reason not to use ortho-rectified products? Are those just much more limited or why else would you not want to use to the non rectified products?

**Comment by gmilcinski** [comment]
Working with non-ortho data will probably cause worse results so it makes sense to use ortho-rectified products.
I can think of the following reasons, why not:
-the response will be a bit slower
-in some areas of the world DEM is not of the best accuracy and might actually distort the data (rare, but I've seen it happening).
I would still use ortho...

**Comment by patrickcgray** [comment]
Okay thanks!!
